source/_posts/2024-10-02-release-202410.markdown

```diff
@@ -0,0 +1,723 @@
+to help each other out. So, join us!
+
+Our very active [Discord chat server](/join-chat) is an excellent place to be
+at, and don't forget to join our amazing [forums](https://community.home-assistant.io/).
+
+Found a bug or issue? Please report it in our [issue tracker](https://github.com/home-assistant/core/issues),
+to get it fixed! Or, check [our help page](/help) for guidance for more
+places you can go.
+
+Are you more into email? [Sign-up for our Building the Open Home Newsletter](/newsletter)
+to get the latest news about features, things happening in our community and
+other news about building an Open Home; straight into your inbox.
+
+## Backward-incompatible changes
+
+We do our best to avoid making changes to existing functionality that might
+unexpectedly impact your Home Assistant installation. Unfortunately, sometimes,
+it is inevitable.
+
+We always make sure to document these changes to make the transition as easy as
+possible for you. This release has the following backward-incompatible changes:
+
+{% details "Climate" %}
+
+We have added more validation to the `climate.set_humidity` action, which
+enforces that the set target humidity is within the targeted entity's minimum
+and maximum humidity levels.
+
+You should adjust automations and scripts to use valid values if you are using
+this action.
+
+([@gjohansson-ST] - [#125242]) ([documentation](/integrations/climate))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#125242]: https://github.com/home-assistant/core/pull/125242
+
+{% enddetails %}
+
+{% details "ecobee" %}
+
+The previously deprecated `aux_heat` attribute has been removed from the
+provided climate entity.
+
+([@gjohansson-ST] - [#125246]) ([documentation](/integrations/ecobee))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#125246]: https://github.com/home-assistant/core/pull/125246
+
+{% enddetails %}
+
+{% details "Honeywell Total Connect Comfort (US)" %}
+
+The previously deprecated `aux_heat` attribute has been removed from the
+provided climate entity.
+
+([@gjohansson-ST] - [#125248]) ([documentation](/integrations/honeywell))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#125248]: https://github.com/home-assistant/core/pull/125248
+
+{% enddetails %}
+
+{% details "Humidifier" %}
+
+We have added more validation to the `humidifier.set_humidity` action, which
+enforces that the target humidity is within the targeted entity's minimum
+and maximum humidity levels.
+
+You might need to adjust automations and scripts to use valid values if you
+are using this action.
+
+([@gjohansson-ST] - [#125863]) ([documentation](/integrations/humidifier))
+
+[@gjohansson-ST]: https://github.com/gjohansson-ST
+[#125863]: https://github.com/home-assistant/core/pull/125863
+
+{% enddetails %}
+
+{% details "Nexia/American Standard/Trane" %}
+
+The `aux_heat` attribute of the climate entity is now deprecated and will be
```

Comment:
This is a deprecation. Maybe ok to mention anyway, but I'm not sure it's intended.